packages/js/product-editor/src/blocks/generic/taxonomy/edit.tsx

```diff
@@ -103,6 +103,10 @@ export function Edit( {
 		value: String( taxonomy.id ),
 	} ) );
 
+	function onClear() {
```

Comment:
I would recommend to use always the pattern `handleClear` or `handleClick` for the event listener function.

Reply:
Done in [2a12d5a](https://github.com/woocommerce/woocommerce/pull/49036/commits/2a12d5a9f2cbb25345bb2a44511e850eb1f11f2d)